README.md

```diff
@@ -1,52 +1,66 @@
 
-Additional Documentation
----
+✨ We constantly upgrade the repo to the latest cordova versions of android, iOS, cordova-lib, and the most recent node and npm versions. The CI will be up-to-date. 
+
+For the latest versions, refer [`package.cordovabuild.json`](https://github.com/e-mission/e-mission-phone/blob/fce117ff859abd995613bd405dbc7d27c703b09b/package.cordovabuild.json)
+
+## Additional Documentation
 Additional documentation has been moved to its own repository [e-mission-docs](https://github.com/e-mission/e-mission-docs). Specific e-mission-phone wikis can be found here:
 https://github.com/e-mission/e-mission-docs/tree/master/docs/e-mission-phone
 
 **Issues:** Since this repository is part of a larger project, all issues are tracked [in the central docs repository](https://github.com/e-mission/e-mission-docs/issues). If you have a question, [as suggested by the open source guide](https://opensource.guide/how-to-contribute/#communicating-effectively), please file an issue instead of sending an email. Since issues are public, other contributors can try to answer the question and benefit from the answer.
 
-Updating the UI only
+:sparkles: Check [Contributing](#contributing) if you're interested in contributing for this project :sparkles:
+
+## Contents
+#### 1. [Updating the UI only](#updating-the-ui-only)
+#### 2. [Updating the e-mission-* plugins or adding new plugins](#updating-the-e-mission--plugins-or-adding-new-plugins)  
+#### 3. [Creating logos](#creating-logos) 
+#### 4. [End to End Testing](#end-to-end-testing)
+#### 5. [Beta-testing debugging](#beta-testing-debugging) 
+#### 6. [Contributing](#contributing)
+#### 7. [Troubleshooting](#troubleshooting)
+
 ---
-[![osx-serve-install](https://github.com/e-mission/e-mission-phone/workflows/osx-serve-install/badge.svg)](https://github.com/e-mission/e-mission-phone/actions?query=workflow%3Aosx-serve-install)
+
+
+## Updating the UI only
+[![osx-serve-install](https://github.com/e-mission/e-mission-phone/workflows/osx-serve-install/badge.svg)](https://github.com/e-mission/e-mission-phone/actions/workflows/serve-install.yml)
 
 If you want to make only UI changes, (as opposed to modifying the existing plugins, adding new plugins, etc), you can use the **new and improved** (as of June 2018) [e-mission dev app](https://github.com/e-mission/e-mission-devapp/) and install the most recent version from [releases](https://github.com/e-mission/e-mission-devapp/releases). 
 
 ### Installing (one-time)
 
-Run the setup script
+1. Run the setup script
 
 ```
-$ bash setup/setup_serve.sh
+bash setup/setup_serve.sh
 ```
-
 **(optional)** Configure by changing the files in `www/json`.
 Defaults are in `www/json/*.sample`
 
 ```
-$ ls www/json/*.sample
-$ cp www/json/startupConfig.json.sample www/json/startupConfig.json
-$ cp ..... www/json/connectionConfig.json
+ls www/json/*.sample
+cp www/json/startupConfig.json.sample www/json/startupConfig.json
+cp ..... www/json/connectionConfig.json
```

Comment:
I have not received any links from either you or Jack. Please revert all these changes; we can include them in a future README change.

Reply:
https://github.com/e-mission/e-mission-phone/pull/1037/files/bbf18bdb939f9a30670cda10c54de4c66a251c84#r1342002279